# Do NICE's recommended questions get answered

## Where did I get this data from?

I went to these urls:
- https://www.nice.org.uk/about/what-we-do/science-policy-research/research-recommendations
- https://web.archive.org/web/20170716190227/https://www.nice.org.uk/about/what-we-do/science-policy-research/research-recommendations

At each respective webpage, I selected everything and then pasted it into VSCode.

Using VSCode I performed the following operations:
1. Removed header and footer. I only wanted the relevant data.
2. Removed the text that follows the tab character. These instances were found using the regex '\t.*\$'
3. Removed the lines that didn't contain the regex '[a-zA-Z]\*\d\*/\d\*'. These were found using the regex '^((?!^[a-zA-Z]\*\d\*/\d\*).)\*\$\n'

The data was then as it is now.

In [18]:
f = open(r"C:\Users\nanki\Documents\Mountain Vault\2017 list of recommended research projects(visual studio code version).md", "r")
list_2017 = f.read().split("\n")

g = open(r"C:\Users\nanki\Documents\Mountain Vault\Current list of recommended research projects(visual studio code version).md", "r")
list_current = g.read().split("\n")

print(len(set(list_2017)))
print(len(set(list_current)))

1067
2103


In [17]:
left_orphans = list(set(list_2017) - set(list_current))
overlap = list(set(list_2017) & set(list_current))
right_orphans = list(set(list_current) - set(list_2017))
print(len(left_orphans))
print(len(overlap))
print(len(right_orphans))

90
977
1126


Of the 1067 unique recommendations given in 2017, only 90 have been removed from the list for any reason. If we assume that all of them were removed because the approprate research was performed, then that still would suggest only 11.6 percent of the questions have been solved.

Given that, of the five examples we have considered above, the two questions which NICE are no longer giving as part of their recommended research were removed having not been answered, it is reasonable to suspect the true rate of question answering is far below 11.6 percent.